neptune_tensorboard/sync/tensorflow_data_sync.py

```diff
@@ -48,8 +48,23 @@ def run(self):
                     except:  # pylint: disable=bare-except
                         pass
 
+    def _init_accummulator(self, path, experiment):
+        accumulator = NeptuneEventAccumulator(path, size_guidance={
+            COMPRESSED_HISTOGRAMS: 0,
+            IMAGES: experiment.limits['channels']['image'],
+            AUDIO: 0,
+            SCALARS: experiment.limits['channels']['numeric'],
+            HISTOGRAMS: 0,
+            TENSORS: experiment.limits['channels']['text']
+        })
+        accumulator.Reload()
+        try:
+            accumulator.FirstEventTimestamp()
```

Comment:
This effectively checks if the file had any events - if not, then the file does not correspond to an experiment.